src/Kovan-DssSpell.sol

```diff
@@ -28,45 +38,255 @@ contract SpellAction {
     // The contracts in this list should correspond to MCD core contracts, verify
     //  against the current release list at:
     //     https://changelog.makerdao.com/releases/kovan/active/contracts.json
+    ChainlogAbstract constant CHANGELOG =
+        ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+
+    /// UNI-A
+    address constant UNI               = 0x0C527850e5D6B2B406F1d65895d5b17c5A29Ce51;
+    address constant MCD_JOIN_UNI_A    = 0xb6E6EE050B4a74C8cc1DfdE62cAC8C6d9D8F4CAa;
+    address constant MCD_FLIP_UNI_A    = 0x6EE8a47eA5d7cF0C951eDc57141Eb9593A36e680;
+    address constant PIP_UNI           = 0xe573a75BF4827658F6D600FD26C205a3fe34ee28;
```

Comment:
This contract is not verified on etherscan.

Reply:
@NiklasKunkel did confirm this is the correct OSM, but it is compiled with solidity `0.6.7` and needs to be verified.  The verification portal is currently down.

Reply:
Just verified: https://kovan.etherscan.io/address/0xe573a75bf4827658f6d600fd26c205a3fe34ee28#code